Confus/Confus/Networking/ClientConnection.cpp

```diff
@@ -53,19 +54,17 @@ namespace Confus
             }
         }
 
-		void ClientConnection::sendMessage(const std::string& a_Message)
+		void ClientConnection::sendMessage(RakNet::BitStream* a_Stream) 
 		{
 			if(m_Connected)
 			{
-				RakNet::BitStream stream;
-				stream.Write(static_cast<RakNet::MessageID>(EPacketType::Message));
-				stream.Write(a_Message.c_str());
-				m_Interface->Send(&stream, PacketPriority::HIGH_PRIORITY,
+				m_Interface->Send(a_Stream, PacketPriority::HIGH_PRIORITY,
 					PacketReliability::RELIABLE_ORDERED, 0, getServerAddress(), false);
```

Comment:
This send message seems to be meant for everything wanting to send message. Problem is, it is now sending RELIABLE_ORDERED messages all the time. Though it should not be that much of a big deal for the client side, I think this should not be set for the user.